lang/java/compiler/src/main/javacc/org/apache/avro/compiler/idl/idl.jj

```diff
@@ -242,6 +242,7 @@ TOKEN :
 | < TIME: "time_ms" >
 | < TIMESTAMP: "timestamp_ms" >
 | < DECIMAL: "decimal" >
+| < LOCAL_TIMESTAMP_MS: "local_timestamp_ms" >
```

Comment:
The other tokens above do not have _MS in the name even if there is a _ms in the string.

Another question: Why are there only tokens for millisec variants but not for microsec variants? This is not only true for the new type but for the old type as well.

Reply:
Make sense, will address this.

Reply:
As of the other question: I was wondering on this question too, and I don't know. The author of the logical types seemed to find unimportant to add IDL representation for microsec, hence I didn't add it either for the new types. So far it seems that nobody else added this either, we can add the new token later if needed (to both types). Actually time type has one precision token too. The spec also has a duration type, which doesn't have any IDL token, though it seems that it isn't even implemented for Java, only for Ruby and C++.